set.mm

```diff
@@ -85685,6 +85710,101 @@ the first case of his notation (simple exponentiation) and subscript it
+       value is a function.  (Contributed by AV, 15-Sep-2024.) $)
+    fsetfcdm $p |- ( X e. A -> S : F --> B ) $=
+      ( wcel cv cfv wf vex feq1 elab2 ffvelrn expcom syl5bi imp fmptd ) GAJZEFG
+      EKZLZBCUBUCFJZUDBJZUEABUCMZUBUFABDKZMUGDUCFENABUHUCOHPUGUBUFABGUCQRSTIUA
+      $.
+
+    $d A f h x $.  $d B h x $.  $d F f h $.  $d S g h $.  $d V f g h x $.
+    $d X f h x $.
+    $( The mapping of an element of a class of functions to a fixed function
+       value is a surjection.  (Contributed by AV, 15-Sep-2024.) $)
+    fsetfocdm $p |- ( ( A e. V /\ X e. A ) -> S : F -onto-> B ) $=
+      ( vh vx wcel wa wf cv cfv wceq adantl cvv wrex wral fsetfcdm simplr fmptd
+      wfo cmpt eqid wb simpll mptexd feq1 elab2g syl mpbird fveq2 fveq1 cbvmptv
+      eqeq2d eqtri a1i fvexd fvmptd eqidd simpr adantr rspcedvd ralrimiva dffo3
+      vex eqtr2d sylanbrc ) AGMZHAMZNZFBCOZEPZKPZCQZRZKFUAZEBUBFBCUFVNVPVMABCDE
+      FHIJUCSVOWAEBVOVQBMZNZVTVQLAVQUGZCQZRZKWDFWCWDFMZABWDOZWCLAVQBWDVOWBLPZAM
+      UDWDUHUEWCWDTMWGWHUIWCLAVQGVMVNWBUJUKABDPZOWHDWDFTABWJWDULIUMUNUOZVRWDRZV
+      TWFUIWCWLVSWEVQVRWDCUPUSSWCWEHWDQZVQWCDWDHWJQZWMFCTCDFWNUGZRWCCEFHVQQZUGW
+      OJEDFWPWNHVQWJUQURUTVAWJWDRWNWMRWCHWJWDUQSWKWCHWDVBVCVOWMVQRWBVOLHVQVQAWD
+      TVOWDVDVOWIHRNVQVDVMVNVEVQTMVOEVJVAVCVFVKVGVHKEFBCVIVL $.
+  $}
+
+  ${
+    $d A a f g m n $.  $d B a f g m n $.  $d V a m n $.
+    $( The class of all functions from a nonempty set ` A ` into a proper class
+       ` B ` is not a set.  If one of the preconditions is not fufilled,
+       ` { f | f : A --> B } ` is a set, see ~ fsetdmprc0 for ` A e/ _V ` ,
+       ~ fset0 for ` A = (/) ` , and ~ fsetex for ` B e. _V ` , see also
+       ~ fsetexb .  (Contributed by AV, 14-Sep-2024.)  (Proof shortened by BJ,
+       15-Sep-2024.) $)
+    fsetprcnex $p |- ( ( ( A e. V /\ A =/= (/) ) /\ B e/ _V )
+                       -> { f | f : A --> B } e/ _V ) $=
+      ( va vg vm vn wcel c0 wne wa cvv wnel cv wf cab wi wex cfv n0 cmpt cbvabv
+      feq1 fveq1 cbvmptv fsetfocdm fornex syl5com nelcon3d expcom exlimiv sylbi
+      wfo impcom imp ) ADIZAJKZLBMNZABCOZPZCQZMNZURUQUSVCRZUREOZAIZESUQVDRZEAUA
+      VFVGEUQVFVDUQVFLZVBMBMVHVBBFVBVEFOZTZUBZUNVBMIBMIABVKGHVBDVEVAABGOZPCGABU
+      TVLUDUCFHVBVJVEHOZTVEVIVMUEUFUGVBBMVKUHUIUJUKULUMUOUP $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The class of all functions from a nonempty set ` A ` into a class ` B `
+       is a set iff ` B ` is a set .  (Contributed by AV, 15-Sep-2024.) $)
+    fsetcdmex $p |- ( ( A e. V /\ A =/= (/) )
+                      -> ( B e. _V <-> { f | f : A --> B } e. _V ) ) $=
+      ( wcel c0 wne wa cvv cv wf cab fsetex wnel wn fsetprcnex ex 3imtr3g con4d
+      df-nel impbid2 ) ADEAFGHZBIEZABCJKCLZIEZABCIMUBUCUEUBBINZUDINZUCOUEOUBUFU
+      GABCDPQBITUDITRSUA $.
+
+    $( The class of all functions from a class ` A ` into a class ` B ` is a
+       set iff ` B ` is a set .  (Contributed by AV, 15-Sep-2024.) $)
```

Comment:
```suggestion
       set iff ` B ` is a set or ` A ` is not a set or ` A ` is empty.  (Contributed by AV, 15-Sep-2024.) $)
```
Also, this theorem looks less important to me. It's borderline Main/mathbox, but non-blocking for me.

Reply:
Done (comment revised, but I prefer it in main, because it summarizes all the different cases within one theorem)

Reply:
BTW, the characterization may be clearer if formulated as
```
$( ... iff, if A is a nonempty set, then B is a set. $)
$p |- ( ... <-> ( A e. ( _V \ { (/) } ) -> B e. _V ) ) $= ? $.
```